dev/Microsoft.Process.Environment/EnvironmentManager.cpp

```diff
@@ -66,18 +66,13 @@ namespace winrt::Microsoft::ProjectReunion::implementation
         }
     }
 
-    void EnvironmentManager::SetEnvironmentVariable(hstring const& name, hstring const& value)
+    bool EnvironmentManager::IsSupported()
     {
-        // If we are running in process scope it is okay to
-       // set EV's because they will not be tracked.
-
-       // Disallow sets if current OS is 20H0 or lower because
-       // tracking EV changes relies on a feature in builds above 20H0
-        if (m_Scope != Scope::Process && IsCurrentOS20H2OrLower())
-        {
-            THROW_HR(E_NOTIMPL);
-        }
+        throw hresult_not_implemented();
```

Comment:
I thought a throw hresult_not_implemeneted was the standard sign of "This needs to be implemented and I haven't done it yet?"

EnvironmentManager will return empty defaults if IsSupported returns false.  This is because it relies on some platform features that are only is 21H1.

Reply:
>I thought a throw hresult_not_implemeneted was the standard sign
Yes. Sorry, I thought you were closer to 'done' than perhaps you are

Reply:
Yeah.  I have a few more things left.
1. AppendToPathExecutable
2. RemoveFromPathExecutable
3. IsSupported.